app/components/form/fields/DisksTableField.tsx

```diff
@@ -67,15 +67,15 @@ export function DisksTableField({
                     <Truncate text={item.name} maxLength={35} />
                   </MiniTable.Cell>
                   <MiniTable.Cell>
-                    <Badge variant="solid">{item.type}</Badge>
+                    <Badge>{item.type}</Badge>
                   </MiniTable.Cell>
                   <MiniTable.Cell>
                     {item.type === 'attach' ? (
                       '—'
                     ) : (
                       <>
                         <span>{bytesToGiB(item.size)}</span>
```

Comment:
Not a change we need to make now, but should we make a size component that ensures the item and the unit are handled consistently / spacing and text colour.